test/verify/check-pages

```diff
@@ -23,6 +23,9 @@ import os
 import parent
 from testlib import *
 
+RHEL_DOC_BASE = ("https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/"
```

Comment:
Maybe we can better just `#noqa` this?

Reply:
I'm not a fan of `#noqa` to second-guess our policy.  However, without the line break the line is 141 characters, so it's actually not critical for *this* PR. And this conflicts with "breaking up URLs is evil", so I'll just re-join this for now.